Chef.PowerShell.Wrapper.Core/Chef.PowerShell.Wrapper.Core.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+typedef void * allocation_function(size_t size);
+
 extern "C" {
-	_declspec(dllexport) const wchar_t* ExecuteScript(const char* powershellScript, int timeout = -1);
-}
+    _declspec(dllexport) const wchar_t* ExecuteScript(const char* powershellScript, int timeout, allocation_function* ruby_allocate);
```

Comment:
with the `-1` default removed, will this affect chef code that does not pass a timeout?

Reply:
Yeah I think this was answered for me further down in the code changes. Thanks